Model/ResourceModel/CsvHsCode.php

```diff
@@ -27,16 +27,28 @@ protected function _construct()
         $this->_init('reach_hs_code', 'id');
     }
 
+    public function testProductDetails($sku) {
```

Comment:
I'm a little worried about this testing code being sprinkled around the code base. I can see where you're going but it might make reading the code later more difficult since the test functions aren't in one place.

Did it help you when doing this code change to have these test methods? I'm good to keep them if I'm missing something

Reply:
Even though the methods say 'test', they are for outputing values to the debug console. Because some classes do not allow the debugger to be imported, these functions pass these values back to where they can be outputed. I would recommend leaving them as they provide valuable debugging info and merchants can have the log level not set to 'debug'.

Reply:
I have the exact same concerns as Lizzy has. In the spirit of good programming practices I guess it is more meaningful to change the names to something like getXYZ(..) . Thanks!